### ISSUE TITLE
Default theme

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5-design#105]
**Note**: This is an *umbrella issue*, a heart of discussion around themes in CKE5. The aim is to avoid fragmentation and confusion about the mockups.

## Overview

CKEditor 5 needs a new, default theme.

1. It must keep up with design trends.
2. It must meet accessibility standards and requirements.
3. It must reflect the flexibility, modularity and extensibility of the editor.
4. It should be easy to integrate with web pages and applications, both visually and technically.

## Related tickets

----

## Mockups

These mockups follow the [Implementations](https://github.com/ckeditor/ckeditor5-design/wiki/Implementations ) document. This is the first attempt to take the [ongoing UX discussion](https://github.com/ckeditor/ckeditor5-design/labels/UX) to the next level, an entry point to further discussion.

### Classic interface

![image](https://cloud.githubusercontent.com/assets/1099479/12611459/ebed0d44-c4ec-11e5-806d-1eee126bdceb.png)

### Tabbed interface

![image](https://cloud.githubusercontent.com/assets/1099479/12611465/f1e19dd2-c4ec-11e5-9f30-fcc1057435e3.png)

### Menu

![image](https://cloud.githubusercontent.com/assets/1099479/12611480/ffe55b44-c4ec-11e5-903d-8a01789ea1e2.png)

### Dialogs

![05-dialogs](https://cloud.githubusercontent.com/assets/1099479/12611503/1fb6d538-c4ed-11e5-8688-959163637453.png)
![06-dialogs](https://cloud.githubusercontent.com/assets/1099479/12611502/1fb22f74-c4ed-11e5-8510-70dbe1e254ad.png)

### Inline editing

![image](https://cloud.githubusercontent.com/assets/1099479/12611516/30077e2e-c4ed-11e5-8de6-93cc1ec49caa.png)

![image](https://cloud.githubusercontent.com/assets/1099479/12611519/398f53ea-c4ed-11e5-853c-d4f71e96d2e4.png)

![image](https://cloud.githubusercontent.com/assets/1099479/12611528/4b167080-c4ed-11e5-84b3-2b97ad76243f.png)

### Fullpage

![11-fullpage](https://cloud.githubusercontent.com/assets/1099479/12611289/303d6a12-c4ec-11e5-8b5a-0068829dffb8.png)

**Comment by Reinmar** [comment]
What are the questions that we want to answer here? I mean – are we meant to discuss (spacing, colors, etc.) the designs already or sth else still?

**Comment by oleq** [comment]
@Reinmar See "Related issues" section.

**Comment by Reinmar** [comment]
OK, changing the title to "The problems to consider" makes this clear.

**Comment by oleq** [comment]
Yes, you were right. Also note that this is the beginning – there will be more problems in the future.

**Comment by koleary** [comment]
Overall very nice. 

Tabs could be even simpler, perhaps just heavy over or underline. 

Dark title bar need not be so dark if tabs have no background. The lighter color can be the top bar background. It looks a bit heavy now.

My personal preference is larger and lighter weight for titles.

**Comment by GeorgePhillips** [comment]
This looks like a big improvement. Have you considered moving the labels above the form fields. This would improve the usability and provide more space for UI elements.

- StackExchange discussion on the topic http://ux.stackexchange.com/questions/44095/which-is-more-usable-labels-above-form-elements-or-to-the-right
- UX argument for above labels http://uxmovement.com/forms/faster-with-top-aligned-labels/
- Material design on labels https://www.google.com/design/spec/components/text-fields.html#text-fields-labels

Keep up the good work

**Comment by FinalAngel** [comment]
The tabbed interface looks great :+1:

**Comment by oleq** [comment]
@GeorgePhillips Good point, thanks!

**Comment by Reinmar** [comment]
I've been just reviewing the dropdown component and found the dropdown styling very confusing. It's so white in relation to totally unmarked buttons that they look like active all the time. I think that the styling should be very similar, if not identical.

See  https://github.com/ckeditor/ckeditor5-theme-lark/pull/12#issuecomment-221545376

**Comment by oleq** [comment]
I don't think so, @Reinmar. It was a conscious design decision. Take a look on the interface of popular text processors like [LibreOffice Writer](https://www.libreoffice.org/discover/screenshots/) or MS Word 2016. It's a common practice that dropdowns and other complex controls in the toolbar are slightly different than the rest of the interface.

**Comment by Reinmar** [comment]
Hm.. you're right. But:

* I'm not sure whether this is necessary in case of a few buttons. 
* I think that you can type in those dropdowns in LO Writer and MS Word, so their white bg may be indicating that they are input fields. Different story. And this may even make our dropdown buttons more confusing.

But most importantly, it just looks odd when you have just couple of buttons. The below screenshot is of course an extreme situation when you have exactly three items and each of them is different. But we should take such case into consideration.

![screen shot 2016-05-25 at 16 53 47](https://cloud.githubusercontent.com/assets/156149/15544656/c6741e94-2299-11e6-9227-69fedf88637c.png)

**Comment by Reinmar** [comment]
BTW when I look at designs like this https://twitter.com/steipete/status/735377624265072641 I start to think that perhaps we should adopt a single convention and follow it closely. I think that perhaps we've been trying to be too innovative. For many aspects that makes sense, and the UX in some sense will be one of them, but design not necessarily.

**Comment by luzfcb** [comment]
sorry if this is the wrong place to ask:

since html has no implements a concept of physical page (A4 paper), ckeditor 5 will provide anything that helps to implement the editor with concept of physical page, like google docs, ms office online do?

**Comment by Reinmar** [comment]
Thanks for asking. That's a topic for a separate ticket: https://github.com/ckeditor/ckeditor5-design/issues/147
